### ISSUE TITLE
Login view config doesn't respect enabled

### ISSUE THREAD

**Issue by erbridge** [stormpath/express-stormpath#106]
I'm trying to implement my own login view and switching `web.login.enabled = false` doesn't disable the login endpoint.

```js
{
  expand: {
    customData: true,
  },

  website: true,

  web: {
    register: {
      enabled:       true,
      autoAuthorize: true,
      fieldOrder: [
        'username',
        'email',
        'givenName',
        'surname',
        'password',
        'passwordConfirm',
      ],
    },
    login: {
      enabled: false,
    },
    logout: {
      enabled: true,
    },
    forgotPassword: {
      enabled: true,
    },
    changePassword: {
      enabled:   true,
      autoLogin: true,
    },
  },
}
```

**Comment by typerandom** [comment]
@erbridge Thank you for reporting this issue.

The issue here is that `website: true` conflicts with the `web` configuration.

What `website: true` actually does it that it enables the `web` endpoints `register`, `login`, `logout` and `me`. What you can do meanwhile is remove `website: true` and add `register` and `me` to your list of enabled `web` endpoints.

I.e. so it would look like this:

```js
{
    expand: {
        customData: true
    },
    web: {
        register: {
            enabled: true,
            autoAuthorize: true,
            fieldOrder: [
                'username',
                'email',
                'givenName',
                'surname',
                'password',
                'passwordConfirm'
            ]
        },
        login: {
            enabled: false
        },
        logout: {
            enabled: true
        },
        forgotPassword: {
            enabled: true
        },
        changePassword: {
            enabled: true,
            autoLogin: true
        },
        me: {
            enabled: true
        },
        register: {
            enabled: true
        },
    }
}
```

Let me know if this helped you.

**Comment by typerandom** [comment]
There's now a PR in our config module that contains a fix for this issue. Once we have merged that in and have updated the integration to use that version, this issue will be resolved.

**Comment by robertjd** [comment]
This should be fixed in 2.1.0, please re-open if that is not the case.
